src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -134,7 +133,6 @@ export default App.extend({
     this.showSortDroplist();
     this.showTypeToggle();
     this.showTableHeaders();
-    this.startFiltersApp();
```

Comment:
why was this change necessary?  Work was done to make sure the filters app was updated as data changed.. so that'd need to come out.. but ideally it could be in onBeforeStart

Reply:
ah... this is because the `filters.type` is changing outside of the filters app.  Ok.. well.. bad news... but I think it makes sense..  we need to move the toggle into the filters app.  Then all of the filters state is controlled by the same app.

Reply:
I changed this back because the first click on a filter worked, but any subsequent click didn't fire off an event

Reply:
I think I get it.. so.. we should remove stateEvents from the filters app.. since those'll never happen.. and let's put this back in `onBeforeStart` and add this to the `if(isRestarting`  I _think_ that'll work and handle state changes during loading

Reply:
nope, no dice. Same behavior as before: first interaction with a filter component changes does what you think it would: sets the state in the store, new requests are fired off, list updates (though the 'reset button' doesn't show up until a page refresh). Then every other interaction with a filter only updates the droplist, but the store isn't being updated and so requests aren't being fired/the list doesn't get updated

Reply:
back to the drawing board I guess

Reply:
Ok.. I'm pretty sure everything works as it was in my commit with the exception of this line which needs to change:
```js
  this.listenTo(filtersApp.getState(), 'change', ({ attributes }) => {
    this.setState({ filters: _.clone(attributes) });
  });
```
The issue is that we're changing an object for the same object so even though the data on the object changed the object referenced did not so no change event was occurring..  Generally why we don't nest state data (and want flat data from endpoints) but I dunno.. seemed cleaner this time :-/